docs/DeveloperGuide.md

```diff
@@ -86,6 +86,12 @@ AddItemCommand command = new AddItemCommand(itemName, quantity, expiryDate);
 command.execute(itemMap, storage);
 ```
 
+<!-- @@author cnivedit -->
+
+#### AddItemCommand
+
+<img src = "diagrams/AddItemCommand-SequenceDiagram.png"/>
```

Comment:
![image](https://github.com/user-attachments/assets/387a2feb-4ec2-4d96-90ff-a0928689e5c3)
Calling themselves here is missing the activation bar.